internals/Cargo.toml

```diff
@@ -18,9 +18,15 @@ default = []
 std = ["alloc"]
 alloc = []
 
+test-serde = ["serde/derive", "serde_json", "bincode"]
```

Comment:
Why does it require `derive`?

Reply:
Oh, that was from when I had the tests (since removed), so this is stale. Will remove.